runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigureTask.java

```diff
@@ -302,7 +303,7 @@ public Void call() throws Exception
     }
 
 
-    private String readURL(
+    public String readURL(
```

Comment:
I think, we need to change it. Currently, it is accepting URL and grpc service can be just a path. What do you think I haven't changed it to get your opinion